Automattic/wp-calypso
# Calypso links: Update links to redirect to developer.wordpress.com

Related to  https://github.com/Automattic/dotcom-forge/issues/5782

## Proposed Changes

* Update links to redirect to developer.wordpress.com, that required a small change on help center since it assumes, until now, that support would be the only source of data. Now you can dynamically pass a blog_id. 

**The issue is mentioning 6 links, but I was only able to find 3.**

 * **Under {LIVE_LINK}/hosting-config/{YOUR_SITE_SLUG}**
<img width="1364" alt="image" src="https://github.com/Automattic/wp-calypso/assets/2653810/70d00beb-ef62-40d2-86e9-b468dea5422d">

<img width="1322" alt="image" src="https://github.com/Automattic/wp-calypso/assets/2653810/55539046-3d5e-4cc3-859f-e19c55353773">

* **Under {LIVE_LINK}/site-monitoring/{YOUR_SITE_SLUG}**
<img width="1426" alt="image" src="https://github.com/Automattic/wp-calypso/assets/2653810/3fb0d8a4-156b-4b4b-9cab-70bc26e02e34">

These links should point to `developer.wordpress.com` instead of support

## Testing Instructions



* Pull and run this branch or use live link.
* Check the images above with the explanation where to find the links
* Check if the links are pointing to developer.wordpress.com 

## Pre-merge Checklist
